### DISCUSSION TITLE
keyboard shortcut: copilot prompts global vs language for disable

### DISCUSSION THREAD

**Discussion by cgbur** [community/community#20469] [Copilot Conversations]
Copilot extension on vscode should have an option to specifically disable either language or global when toggling, as well as a general disable option to not break current users. Currently trying to disable the extension prompts the user to use the mouse to select an option. This makes using a keyboard shortcut to quickly toggle copilot useless.

**Comment by campbellcole** [comment]
I worked around this by making a macro on my keyboard to execute the shortcut then press enter. I should not have to do that. I agree that this shortcut, as is, is completely useless.
